src/test/regress/expected/failure_mx_metadata_sync_multi_trans.out

```diff
@@ -370,6 +371,23 @@ SELECT citus.mitmproxy('conn.onQuery(query="CREATE TABLE mx_metadata_sync_multi_
 
 (1 row)
 
+SELECT citus_activate_node('localhost', :worker_2_proxy_port);
+ERROR:  connection to the remote node localhost:xxxxx failed with the following error: connection not open
+-- Failure to record sequence dependency for table
+SELECT citus.mitmproxy('conn.onQuery(query="SELECT pg_catalog.worker_record_sequence_dependency").cancel(' || :pid || ')');
```

Comment:
I'd be intereseted in seeing failures at the other stages of creation of  `dist1`.

Do you mind adding some more cases?